code/datums/emotes.dm

```diff
@@ -90,6 +92,16 @@
 		user.emotes_used = list()
 	user.emotes_used[src] = world.time
 	return TRUE
+	
+/datum/emote/proc/check_audio(mob/living/user, intentional)
```

Comment:
This code does not meet our coding standards for dmdoc documentation. [More information on our documentation requirements can be found here](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md)

Reply:
Actually I have reconsidered and I think you could consider this an internal function. To this extent I would still *suggest* documenting it, as it would take little effort and make your code more maintainable.